plugins/SliceInfoPlugin/SliceInfo.py

```diff
@@ -119,6 +119,7 @@ def _onWriteStarted(self, output_device):
             }
             for container in global_container_stack.getContainers():
                 container_id = container.getId()
+                container_serialized = ""
```

Comment:
Yes, but shouldn't the code throw at `if container_serialized` an error which says that `container_serialized` is undefined?
I'm a bit confused now.. or am I overlooking something here?
And for example, what if the 3 container isn't serializable again? It won't update `container_serialized` and fill it to the dict with the id of the new (third) container... 🤔